### PR TITLE
samples: update gfe_latency sample

### PR DESCRIPTION
**Pull request by KiranmayiB**
Updates Cloud Spanner opencensus samples for capturing GFE latencies

### CODE REVIEW

**Review by thiagotnunes — APPROVED** [review]

**Comment by thiagotnunes** [comment]
Please update your description. Also please update your title to match https://www.conventionalcommits.org/en/v1.0.0/ (something like `samples: update gfe latency sample`)

**Comment by KiranmayiB** [comment]
DO NOT merge before November 26th 2021. Waiting for an internal approval.

**Review by lesv — APPROVED** [review]

**Pull request merged**
